### PR TITLE
BUG: cleanup unnecessary warning

### PR DESCRIPTION
**Pull request by bsipocz**
Fixes https://github.com/astropy/sphinx-astropy/issues/70, see discussion there for context and current behaviour.

### CODE REVIEW

File: sphinx_astropy/conf/v1.py [hunk 1]
```diff
@@ -217,10 +217,11 @@ def check_sphinx_version(expected_version):
 # AttributeError is checked here in case matplotlib is installed but
 # Sphinx isn't.  Note that this module is imported by the config file
 # generator, even if we're not building the docs.
+# We don't need to raise a warning or exception here as there are packages
+# that don't use the directive, and those who try to use it without mpl being
+# installed will get a nice "Unknonw directive type" error at usage.
```
Comment (pllim) [inline sphinx_astropy/conf/v1.py:222]:
```suggestion
# installed will get a nice "Unknown directive type" error at usage.
```

File: sphinx_astropy/conf/v2.py [hunk 2]
```diff
@@ -208,10 +207,11 @@
 # AttributeError is checked here in case matplotlib is installed but
 # Sphinx isn't.  Note that this module is imported by the config file
 # generator, even if we're not building the docs.
+# We don't need to raise a warning or exception here as there are packages
+# that don't use the directive, and those who try to use it without mpl being
+# installed will get a nice "Unknonw directive type" error at usage.
```
Comment (pllim) [inline sphinx_astropy/conf/v2.py:212]:
```suggestion
# installed will get a nice "Unknown directive type" error at usage.
```

**Review by pllim — COMMENTED** [review]
Found small typo. Once that is fixed, I will approve. Thanks!

**Comment by pllim** [comment]
Do you need a release?

**Comment by bsipocz** [comment]
nope, it can wait.

**Pull request merged**